### ISSUE TITLE
AVT 1 - Accordion Component DAP violations on the Select input field

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4522]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component
Run DAP and the following violations are present (see screenshot):
<img width="1219" alt="Screen Shot 2019-11-01 at 11 12 20 AM" src="https://user-images.githubusercontent.com/21676914/68038688-85b57100-fc98-11e9-80c4-00d30874b963.png">

**Comment by dakahn** [comment]
@snidersd i'm testing over at http://react.carbondesignsystem.com/?path=/story/accordion--default and I'm unable to re-create these DAP errors. Turned on 2.1, September, and Experimental rulesets just to check and still get nothing. 

I'm testing on Chrome latest in Windows 10 fwiw

**Comment by snidersd** [comment]
@dakahn I just retested and I'm not seeing any violations either. It looks like this may have been fixed in #4450.
